docs/content/v0.20/packaging.md

```diff
@@ -0,0 +1,28 @@
+# Packaging and Delivery
+
+## status-addon
+
+The [status-addon](https://github.ibm.com/dettori/status-addon) repo is the source of a RedHat-style operator. 
+
+The operator is delivered by a Helm chart at `quay.io/pdettori/status-addon-chart`. Chart versioning? How is the chart built, pushed?
+
+There are probably one or more container images involved.
+
+## KubeStellar
+
+### Central container image
+
+Appears to be at `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`. `make ko-build-push` will create that image. `make ko-build-local` will make a local image for just the local platform.
```

Comment:
Maybe we can say "current snapshot available at... " - "appears to be" seems to indicate something we are guessing about, but we do know

Reply:
Revised.